modular-docs-manual/content/topics/module_guidelines-concept.adoc

```diff
@@ -33,6 +33,37 @@ Do not include any instructions to perform an action, such as executing a comman
 Action items belong in procedure modules.
 See also link:http://www.informationmapping.com/fspro2013-tutorial/infotypes/infotype2.html[The Six Information Types] at _informationmapping.com_ for ways to present different types of conceptual information: principle, concept, structure, process, fact.
 
+If the concept module is large and complex, consider splitting the concept module into multiple standalone concept modules. If you cannot split the module into meaningful standalone modules, consider using subheadings in the module to structure the content for improved reader navigation. If you use subheadings in a concept module, you can add the `[discrete]` tag to exclude each subheading from the table of contents, if needed. However, in many cases, subheadings are helpful to include in the table of contents to improve content searchability.
```

Comment:
```suggestion
If the concept module is large and complex, consider splitting the concept module into multiple standalone concept modules. If you cannot split the module into meaningful standalone modules, consider using subheadings in the module to structure the content for improved reader navigation. If you use subheadings in a concept module, you can add the `[discrete]` tag to render subheadings as sections that are not included in the table of contents. However you should consider if this negatively affects content searchability. In most cases you want to include subheadings in the table of contents so readers can more easily scan.
```

Reply:
@oraNod In terms of minimalism, I feel like that proposed rewording is a wordier way of saying the same thing. Is there something specific that's missing that you think I should be adding? I don't see anything new in the rewording, just an expanded, less concise version of the same ideas. If I'm crazy, please let me know :)

Reply:
ok, disregard @sterobin